### ISSUE TITLE
Maintain compatability with Tor

### ISSUE THREAD

**Issue by DamonGant** [IndependentOnion/rotor#1]
This is a very important thing to get right. I've been outspoken on Twitter about how I think this project is idiotic and adds unneeded fragmentation, but there is one thing I'm particularly concerned about: **Avoiding splitting the network.**

There are actual activists and journalists relying on Tor to protect their identity. While correlation attacks were never meant to be fully adverted by Tor, is probably is the best solution for avoiding sophisticated deanonymization attempts. The number of peers on the network is directly proportional to the amount of effort you need to put in to do this.

Under no circumstances should there be a change in this forks that renders one unable to use the peers of the other (and vice-versa), unless one is the clear winner in user count. This should be easy enough to measure via version string reported to the dirauth should it ever come to this decision.

**Comment by indornet** [comment]
Yes we agree with you here. We are planning on re-appropriating the existing Tor node infrastructure through scraping network descriptors. We'll then add those into our directory authorities.

Thanks for your input!

**Comment by DamonGant** [comment]
It should be relatively easy to both notify and pull peers from a Tor DirAuth _and_ one custom set. This really should not happen inside a blackbox on some server. While reporting to Tor should be togglable, this should be on by default when routing (=relaying) is enabled.
